plenario/api/validator.py

```diff
@@ -54,13 +66,16 @@ class NoGeoJSONValidator(Validator):
     response format."""
 
     valid_formats = {'csv', 'json'}
+    # Validator re-initialized so that it doesn't use old valid_formats.
+    data_type = fields.Str(default='json', validate=OneOf(valid_formats))
```

Comment:
validator wasn't checking the formats correctly, because the data_type validator would be initialized with the old set of valid_formats. You could still give it 'geojson'